specification/resources/resource-manager/readme.md

```diff
@@ -64,6 +64,15 @@ tag: package-deploymentscripts-2020-10
 tag: package-2020-09
```

Comment:
This tag doesn't have details - looking at the file history the tag got renamed by another PR it seems like. Not sure if this is creating any issues while running autorest.

Reply:
But wouldn't that have caused the earlier PR also to have the FileLint/Avocado issues?

Reply:
I think that fixed the build issues with .NET SDK.